source/gui/settingsDialogs.py

```diff
@@ -1977,6 +1977,24 @@ def makeSettings(self, settingsSizer):
 		self.bindHelpEvent("KeyboardSettingsHandleKeys", self.handleInjectedKeysCheckBox)
 		self.handleInjectedKeysCheckBox.SetValue(config.conf["keyboard"]["handleInjectedKeys"])
 
+		minDelay = int(
+			config.conf.getConfigValidation(("keyboard", "maxRepeatedKeyPressDelay")).kwargs["min"]
```

Comment:
Same thing here (and in other places). Even if the rest of it stays the same, I would recommend "timeout" over "delay".